csp/impl/types/common_definitions.py

```diff
@@ -62,6 +66,30 @@ def __init__(self, *args, **kwargs):
         ...
 
 
+def _make_pydantic_outputs(kwargs):
+    """Add pydantic functionality to Outputs, if necessary"""
+    from pydantic import create_model
+    from pydantic_core import core_schema
+
+    from csp.impl.wiring.outputs import OutputsContainer
+
+    if None in kwargs:
+        typ = ContainerTypeNormalizer.normalize_type(kwargs[None])
+        model_fields = {"out": (typ, ...)}
```

Comment:
how are model field names used?

Reply:
Because we are just using the output type for validation - the name of the field is not relevant. All we check is that it validates [here](https://github.com/Point72/csp/pull/179/files#diff-a3fa112acbdbdf692f3b09e27cb8b49cc1cea655db2670f5d549082a72440db8R103)